### PR TITLE
feat: added content with utility, language and registration button

### PR DESCRIPTION
**Pull request by MFelipeSilva**

### CODE REVIEW

File: packages/ui/Button.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import { styled } from "styled-components";
+
+interface ButtonProps {
+  setPosition: string;
+  setIndex: string;
+};
+
+export const UIButton = styled.button<ButtonProps>`
```
Comment (RichardPompeo) [inline packages/ui/Button.ts:8]: @MFelipeSilva please choose a better name to the button

**Review by RichardPompeo — CHANGES_REQUESTED** [review]

File: apps/web/components/UtilityContent.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  const { t } = useTranslation();
+
+  return (
+    <Container>
+      <UIButton setIndex="none" setPosition="none">
+        <RiGlobeFill fontSize={23} />
+      </UIButton>
+      <ContentRegistration>
+        <ButtonSI>{t("utility.buttonsi")}</ButtonSI>
+        <ButtonSU>{t("utility.buttonsu")}</ButtonSU>
```
[the remark this answers is not in the document]
Comment (MFelipeSilva) [reply apps/web/components/UtilityContent.tsx]: mudanças realizadas

[the remark this answers is not in the document]
Comment (MFelipeSilva) [reply apps/web/components/UtilityContent.tsx]: changes made.

File: apps/web/languages/en.json [hunk 3]
```diff
@@ -18,6 +18,10 @@
         "title": "Have your own <0>person space</0>, and use it however your want.",
         "subtitle": "Create an account to save and store everything you do on the site.",
         "aboutTheSiteButton": "About the site"
+      },
+      "utility": {
+        "buttonsi": "Sign in",
+        "buttonsu": "Sign up"
       }
```
[the remark this answers is not in the document]
Comment (MFelipeSilva) [reply apps/web/languages/en.json]: changes made.

File: apps/web/languages/pt.json [hunk 4]
```diff
@@ -18,6 +18,10 @@
         "title": "Tenha seu próprio <0>espaço pessoal</0> e use-o como quiser.",
         "subtitle": "Crie sua conta para salvar e armazenar tudo que você fizer.",
         "aboutTheSiteButton": "Sobre esse site"
+      },
+      "utility": {
+        "buttonsi": "Entrar",
+        "buttonsu": "Inscrever-se"
```
[the remark this answers is not in the document]
Comment (MFelipeSilva) [reply apps/web/languages/pt.json]: changes made.

File: apps/web/styles/components/UtilityStyles.ts [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  box-shadow: 4px 5px black;
+  background-color: rgba(142, 181, 240);
+
+  @media (max-width: 1200px) {
+    padding: 0.6em 1.6em;
+    box-shadow: 2px 3px;
+  }
+`;
+
+export const ButtonSI = styled(ButtonSU)`
```
[the remark this answers is not in the document]
Comment (MFelipeSilva) [reply apps/web/styles/components/UtilityStyles.ts]: changes made.

File: packages/ui/Button.ts (hunk 1 above)
Comment (MFelipeSilva) [reply packages/ui/Button.ts]: changes made.

File: apps/web/styles/components/UtilityStyles.ts [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+`;
+
+export const ContentRegistration = styled.div`
+  display: flex;
+  gap: 1em;
+  padding-left: 30px;
+  border-left: 1px solid ${({ theme }) => theme.COLORS.DEFAULT};
+`;
+
+export const RegisterButton = styled.button`
```
Comment (RichardPompeo) [inline apps/web/styles/components/UtilityStyles.ts:20]: @MFelipeSilva please change this button name

**Review by RichardPompeo — CHANGES_REQUESTED** [review]

File: apps/web/components/UtilityContent.tsx [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+          >
+            <BsGearFill fontSize={23} />
+          </UtilityResponsiveButton>
+        </Popover>
+        <ContentRegistration>
+          <CustomSecondaryButton
+            setColor="#ffffff"
+            setColorBorder="#26262c"
+            setBoxShadow="none"
+            setBackgroundColor="#26262c"
```
Comment (RichardPompeo) [inline apps/web/components/UtilityContent.tsx:78]: @MFelipeSilva what is it?

File: apps/web/components/UtilityContent.tsx [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+          >
+            {t("utility.signInButton")}
+          </CustomSecondaryButton>
+          <CustomPrimaryButton
+            setPadding="0.9em 2em"
+            setBackgroundColor="#8EB5F0"
+            setResPadding="0.6em 1.6em"
```
Comment (RichardPompeo) [inline apps/web/components/UtilityContent.tsx:85]: @MFelipeSilva what is it?

File: apps/web/languages/en.json [hunk 9]
```diff
@@ -18,6 +18,11 @@
         "title": "Have your own <0>person space</0>, and use it however your want.",
         "subtitle": "Create an account to save and store everything you do on the site.",
         "aboutTheSiteButton": "About the site"
+      },
+      "utility": {
+        "signInButton": "Sign in",
+        "signUpButton": "Sign up",
+        "popoverTitle": "Settings"
```
Comment (RichardPompeo) [inline apps/web/languages/en.json:25]: @MFelipeSilva I think is better to create another object inside utility object, ex. `"popover": { "title": "Settings" }`, what u think?

File: apps/web/languages/pt.json [hunk 10]
```diff
@@ -18,6 +18,11 @@
         "title": "Tenha seu próprio <0>espaço pessoal</0> e use-o como quiser.",
         "subtitle": "Crie sua conta para salvar e armazenar tudo que você fizer.",
         "aboutTheSiteButton": "Sobre esse site"
+      },
+      "utility": {
+        "signInButton": "Entrar",
+        "signUpButton": "Inscrever-se",
+        "popoverTitle": "Configurações"
```
Comment (RichardPompeo) [inline apps/web/languages/pt.json:25]: @MFelipeSilva I think is better to create another object inside utility object, ex. `"popover": { "title": "Settings" }`, what u think?

File: apps/web/layout/index.tsx [hunk 11]
```diff
@@ -40,7 +42,17 @@ export default function Layout() {
...
                 <SubTitle>{t("home.subtitle")}</SubTitle>
-                <Button>{t("home.aboutTheSiteButton")}</Button>
+                <CustomPrimaryButton
+                  setPadding="1.3em 2em"
+                  setFontSize="clamp(10pt, 1vw, 12pt)"
+                  setFontWeight="600"
+                  setBackgroundColor="#ffffff"
+                  setResFontSize="clamp(9pt, 0.5vw, 10pt)"
+                  setResPadding="0.9em 1em"
+                  setWidth="100%"
```
Comment (RichardPompeo) [inline apps/web/layout/index.tsx:52]: @MFelipeSilva what is it?

**Review by RichardPompeo — CHANGES_REQUESTED** [review]

File: apps/web/styles/components/UtilityStyles.ts (hunk 6 above)
Comment (MFelipeSilva) [reply apps/web/styles/components/UtilityStyles.ts]: changed

File: apps/web/styles/components/UtilityStyles.ts [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+  box-shadow: 4px 5px black;
+  background-color: rgba(142, 181, 240);
+
+  @media (max-width: 1200px) {
+    padding: 0.6em 1.6em;
+    box-shadow: 2px 3px;
+  };
+`;
+
+export const LoginButton = styled(RegisterButton)`
```
[the remark this answers is not in the document]
Comment (MFelipeSilva) [reply apps/web/styles/components/UtilityStyles.ts]: changed

File: apps/web/components/UtilityContent.tsx (hunk 7 above)
Comment (MFelipeSilva) [reply apps/web/components/UtilityContent.tsx:78]: props

File: apps/web/components/UtilityContent.tsx (hunk 8 above)
Comment (MFelipeSilva) [reply apps/web/components/UtilityContent.tsx:85]: props

File: apps/web/layout/index.tsx (hunk 11 above)
Comment (MFelipeSilva) [reply apps/web/layout/index.tsx:52]: props

File: apps/web/languages/en.json (hunk 9 above)
Comment (MFelipeSilva) [reply apps/web/languages/en.json]: corrected

File: apps/web/languages/pt.json (hunk 10 above)
Comment (MFelipeSilva) [reply apps/web/languages/pt.json]: corrected

**Review by RichardPompeo — APPROVED** [review]

**Pull request merged**